mvp-gate-party/src/main/kotlin/cz/tyckouni/mvpgate/party/grpc/SpServiceGrpcImpl.kt

```diff
@@ -32,6 +33,35 @@ class SpServiceGrpcImpl(
         return sp
     }
 
+    override suspend fun create(request: CreateRequest): SpGrpc {
+        LOGGER.debug("Call to SpService.create: {}", request)
+
+        val sp = converter.toGrpc(spService.create(request.name))
+
+        return sp
+    }
+
+    override suspend fun delete(request: ByGuidRequest): Empty {
+        LOGGER.debug("Call to SpService.delete: {}", request)
+
+        try {
+            spService.delete(UUID.fromString(request.guid))
+        } catch (e: Exception) {
+            LOGGER.debug(e.stackTraceToString())
+            throw e
+        }
```

Comment:
I think Peter had this try-catch only for debugging. I would remove it.

Reply:
True, removed.